examples/prerecorded/callback/callback/main.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 Deepgram SDK contributors. All Rights Reserved.
+# Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+# SPDX-License-Identifier: MIT
+
+import os
+from dotenv import load_dotenv
+import logging, verboselogs
+from datetime import datetime, timedelta
+
+from deepgram import (
+    DeepgramClient,
+    DeepgramClientOptions,
+    PrerecordedOptions,
+    FileSource,
+    UrlSource,
+)
+
+load_dotenv()
+
+AUDIO_FILE = "preamble.wav"
+CALL_BACK_URL = (
+    "https://127.0.0.1:8000"  # TODO: MUST REPLACE WITH YOUR OWN CALLBACK ENDPOINT
+)
+
+
+def main():
+    try:
+        # STEP 1 Create a Deepgram client using the API key in the environment variables
+        config = DeepgramClientOptions(
+            verbose=logging.SPAM,
+        )
+
+        deepgram = DeepgramClient("", config)
+
+        # STEP 2 Call the transcribe_file method on the prerecorded class
+        with open(AUDIO_FILE, "rb") as file:
+            buffer_data = file.read()
+
+        payload: FileSource = {
+            "buffer": buffer_data,
+        }
+        # payload: UrlSource = {
+        #     "url": "https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav"
+        # }
+
+        options = PrerecordedOptions(
+            model="nova-2",
+            smart_format=True,
+            utterances=True,
+            punctuate=True,
+        )
+
+        response = deepgram.listen.prerecorded.v("1").transcribe_file_callback(
+            payload, CALL_BACK_URL, options=options
+        )
+        # response = deepgram.listen.prerecorded.v("1").transcribe_url_callback(
+        #     payload, CALL_BACK_URL, options=options
+        # )
```

Comment:
Can delete this commented-out duplicate block?

Reply:
I put that there so people using this can also do this without using a local file.